src/parser/jobs/nin/Duality.js

```diff
@@ -0,0 +1,58 @@
+import React, {Fragment} from 'react'
+//import {Icon, Message} from 'semantic-ui-react'
+
+import {ActionLink} from 'components/ui/DbLink'
+import ACTIONS from 'data/ACTIONS'
+//import STATUSES from 'data/STATUSES'
+import Module from 'parser/core/Module'
+import {Suggestion, SEVERITY} from 'parser/core/modules/Suggestions'
+
+const DUALITY_GCDS = {
```

Comment:
Maybe consider using a whitelist? Something like

```js
const CORRECT_DUALITY = [
	ACTIONS.AEOLIAN_EDGE.id
]

// then you can do...

if (!CORRECT_DUALITY.includes(abilityId)) {
	this._badDualityUses++
}
```

Reply:
The problem with that approach is that it would increment `_badDualityUses` on abilities that don't actually consume the buff (e.g. other OGCDs or Death Blossom). I set it up the way I did specifically because it's only a subset of abilities that actually work with Duality. That said, it occurs to me now that I may have missed one - gotta test whether Duality works on Throwing Dagger. I'll add another comment when I verify that.

Reply:
Ahhhh, of course. Good point.

Reply:
Confirmed Throwing Dagger consumes Duality, so that'll be on the list when I update the PR.